usaspending_api/references/management/commands/load_disaster_emergency_fund_codes.py

```diff
@@ -0,0 +1,182 @@
+import logging
+import re
+
+from collections import namedtuple
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from psycopg2.extras import execute_values
+from psycopg2.sql import SQL
+from usaspending_api.common.csv_helpers import read_csv_file_as_list_of_dictionaries
+from usaspending_api.common.etl import ETLTable, mixins, ETLTemporaryTable
+from usaspending_api.common.etl.operations import insert_missing_rows, update_changed_rows
+from usaspending_api.common.helpers.sql_helpers import get_connection
+from usaspending_api.common.helpers.timing_helpers import ConsoleTimer as Timer
+
+
+DEF_CODE_PATTERN = re.compile("[a-zA-Z0-9]")
+
+CREATE_TEMP_TABLE = """
+    drop table if exists temp_load_disaster_emergency_fund_codes;
+
+    create temporary table temp_load_disaster_emergency_fund_codes (
+        row_number int,
+        code text,
+        public_law text,
+        title text,
+        group_name text
+    );
+"""
+
+logger = logging.getLogger("console")
+
+DisasterEmergencyFundCode = namedtuple(
+    "DisasterEmergencyFundCode", ["row_number", "code", "public_law", "title", "group_name"]
+)
+
+
+class Command(mixins.ETLMixin, BaseCommand):
+    """
+    Used to load DEF Codes either from URI or a local def_code.csv file. There is a tas_list file inside
```

Comment:
Since this is reference data, it should be added to usaspending_api/references/management/commands/load_reference_data.py

Reply:
Added it